aspnetcore/signalr/configuration.md

```diff
@@ -32,6 +33,36 @@ In the .NET client, the same `AddJsonProtocol` extension method exists on [HubCo
 // At the top of the file:
 using Microsoft.Extensions.DependencyInjection;
 
+// When constructing your connection:
+var connection = new HubConnectionBuilder()
+    .AddJsonProtocol(options => {
+        options.PayloadSerializerOptions.PropertyNamingPolicy = null;
+    })
+    .Build();
+```
+
+::: moniker-end
+
+::: moniker range="= aspnetcore-2.2"
+
+JSON serialization can be configured on the server using the [AddJsonProtocol](/dotnet/api/microsoft.extensions.dependencyinjection.jsonprotocoldependencyinjectionextensions.addjsonprotocol) extension method, which can be added after [AddSignalR](/dotnet/api/microsoft.extensions.dependencyinjection.signalrdependencyinjectionextensions.addsignalr) in your `Startup.ConfigureServices` method. The `AddJsonProtocol` method takes a delegate that receives an `options` object. The [PayloadSerializerSettings](/dotnet/api/microsoft.aspnetcore.signalr.jsonhubprotocoloptions.payloadserializersettings) property on that object is a JSON.NET `JsonSerializerSettings` object that can be used to configure serialization of arguments and return values. See the [JSON.NET documentation](https://www.newtonsoft.com/json/help/html/Introduction.htm) for more details.
+ 
+As an example, to configure the serializer to use "PascalCase" property names, instead of the default "camelCase" names, use the following code:
```

Comment:
```suggestion
As an example, to configure the serializer to use "PascalCase" property names, instead of the default "camelCase" names, use the following code in `Startup.ConfigureServices`:
```